frank-flow/src/main/java/org/ibissource/frankflow/api/DirectoryApi.java

```diff
@@ -103,7 +103,10 @@ public Response deleteDirectory(@PathParam("name") String configurationName, @Qu
         if(file.delete()) {
 			return Response.status(Response.Status.OK).build();
         } else {
-			throw new ApiException("unable to remove file ["+path+"]");
+            if (file.listFiles().length > 0) {
+                throw new ApiException("Can't delete folder '"+path+"' with files. Please remove the files first.");
+            }
+			throw new ApiException("Unable to remove file ["+path+"]");
```

Comment:
It seems that the indentation is a bit wrong.

Reply:
It would seem that github messes with spaces and tabs. In my file the indentation is fine but i'll convert all the tabs to spaces in this file